components/behaviors/time-series.schema.json

```diff
@@ -61,20 +61,20 @@
             "pushTracking.customAction": "Push Tracking Custom Action",
             "listOpearation.removeFromList": "List Operation Remove From List",
             "listOperation.addToList": "List Operation Add To List",
-            "leadOperation.changeScore": "Lead Operation Change Score",
-            "leadOperation.campaignStatusInProgression": "Lead Operation Campaign Status In Progression",
+            "leadoperation.scoreChanged": "Lead Operation Score Changed",
+            "leadoperation.statusInProgramProgressionChanged": "Lead Operation Status In Program Progression Changed",
```

Comment:
Just confirmed with Liz to switch name using Campaign instead of Program: statusInCampaignProgressionChanged